install/kubernetes/cilium/templates/validate.yaml

```diff
@@ -66,9 +66,15 @@
   {{- end }}
 {{- end }}
 
+{{- if .Values.authentication.mutual.spire.enabled }}
+  {{- if not .Values.authentication.enabled }}
+    {{ fail "SPIRE integration requires .Values.authentication.enabled=true" }}
+  {{- end }}
+{{- end }}
+
 {{- if .Values.authentication.mutual.spire.install.enabled }}
-  {{- if not .Values.authentication.mutual.spire.enabled }}
-    {{ fail "SPIRE installation requires .Values.auth.mTLS.enabled=true and .Values.auth.mTLS.spire.enabled=true" }}
+  {{- if or ( not .Values.authentication.enabled ) ( not .Values.authentication.mutual.spire.enabled ) }}
+    {{ fail "SPIRE installation requires .Values.authentication.enabled=true and .Values.authentication.mutual.spire.enabled=true" }}
```

Comment:
this was part of https://github.com/cilium/cilium/pull/26167 as well.

Reply:
ah thanks, i guess i'll wait and rebase

Reply:
@sayboras i removed these changes. we should be safe to merge the two PRs without any conflicts :pray:

Reply:
it's great idea :+1:

Reply:
saw that you're PR has been merged at the time of writing :heavy_check_mark:

Reply:
yeah, I am pretty much trying to push all pending PRs, so that we don't need to rush tmr :D